iphone/Classes/TiUIWebView.m

```diff
@@ -83,7 +83,9 @@ - (WKWebView *)webView
     _willHandleTouches = [TiUtils boolValue:[[self proxy] valueForKey:@"willHandleTouches"] def:YES];
 
     _webView = [[WKWebView alloc] initWithFrame:[self bounds] configuration:config];
-
+#if TARGET_OS_SIMULATOR
+    _webView.inspectable = YES;
```

Comment:
The API is only available on iOS 16.4+, so you also need the iOS version check here, otherwise it will crash on older devices

Reply:
ah, good point. I've added the version check from the other PR. Will have to check it on my Mac later